unity/Assets/0Refactored/scenes/laboratory/Laboratory.pc.unity

```diff
@@ -112,44 +112,6 @@ NavMeshSettings:
     debug:
```

Comment:
Entering into experiments doesn't work.
Please add PC_EnterExperiment/VR_EnterScene to the experiment stations when they are generated.